planetcantile/data/generate.py

```diff
@@ -132,15 +132,14 @@ def __post_init__(self):
         pass
         #print(f'Could not find authority for {crs_obj.to_wkt()}')
 
-
-for tmsp in crss:
+for tmsp in crss[10:]:
     # create the tms object
     tms = morecantile.TileMatrixSet.custom(**asdict(tmsp))
-    tmsj = tms.json(exclude_none=True)
-    tmsj = tmsj.replace(CRS_to_uri(tms.supportedCRS), CRS_to_urn(tms.supportedCRS), 1)
-    tmsj = tmsj.replace(CRS_to_uri(tms.boundingBox.crs), CRS_to_urn(tms.boundingBox.crs), 1)
-    _d = json.loads(tmsj)  # get to pretty printed json
+    tmsj = tms.dict()
+    # Include URN to the planetary projections; _geographic_crs is needed by downstream libs, e.g., morecantile 
+    tmsj['supportedCRS'] = tmsj['boundingBox']['crs'] = CRS_to_urn(tms.supportedCRS)
```

Comment:
okay so I am not sure if this correct behavior yet as I can see that the scales and boundingbox crss are different then how I had them defined in the diff for this pr. I think I decided to always specify the bounding boxes in the `naifid00` coordinates and this pr changes that behavior. Some tests should be added to verify that the correct scale set is being set and verify that the correct boundingbox crs is used